src/main/java/com/joyent/manta/monitor/CustomPrometheusCollector.java

```diff
@@ -116,6 +116,64 @@ private void addElapsedTimeMetric(final ImmutableList.Builder<MetricFamilySample
                 value)));
     }
 
+    private void importSocketTimeOutExceptionMetric(final ImmutableList.Builder<MetricFamilySamples> builder) {
```

Comment:
What will the code do if the metric is unavailable? These JMX entries don't show up until the exceptions happen.

Reply:
I think it is ok if we report a count to Prometheus indicating a value of 0 when these exceptions metrics are missing.